iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java

```diff
@@ -107,43 +109,61 @@ public static void cleanUpAfterClass()
 
     @Test
     public void getDigitalTwin() {
-        BasicDigitalTwin getResponse = this.digitalTwinClient.getDigitalTwin(deviceId, BasicDigitalTwin.class);
-        assertEquals(getResponse.getMetadata().getModelId(), E2ETestConstants.MODEL_ID);
-    }
-
-    @Test
-    public void getDigitalTwinWithResponse() {
-        String digitalTwinId = "";
-    }
+        // act
+        BasicDigitalTwin response = digitalTwinClient.getDigitalTwin(deviceId, BasicDigitalTwin.class);
+        ServiceResponseWithHeaders<BasicDigitalTwin, DigitalTwinGetHeaders> responseWithHeaders = digitalTwinClient.getDigitalTwinWithResponse(deviceId, BasicDigitalTwin.class);
```

Comment:
I see so the idea here is testing both get overloads in same test?

Reply:
I thought it would reduce number of devices we create in the e2e. It should help scale better. What do you think?